node-notes-cli/app.js

```diff
@@ -0,0 +1,45 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+async function parse() {
+  const file = await readFile('data.json');
+  const parse = await JSON.parse(file);
+  return parse;
+}
+
+const data = await parse();
+
+async function notes() {
+  const list = data.notes;
+  console.log(list);
+}
+async function create(newNotes) {
+  const key = data.nextId;
+  data.notes[key] = newNotes;
+  data.nextId++;
+  const newObj = JSON.stringify(data, null, 2);
+  await writeFile('data.json', newObj);
```

Comment:
You repeat these two lines of code over and over. It would be nice to not have that duplication!